dockstore-integration-testing/src/test/java/io/dockstore/client/cli/BaseIT.java

```diff
@@ -66,6 +85,156 @@ public class BaseIT {
     public static final String BIOWORKFLOW = WorkflowSubClass.BIOWORKFLOW.toString();
     public static final String APPTOOL = WorkflowSubClass.APPTOOL.toString();
 
+    @SuppressWarnings("checkstyle:parameternumber")
+    public static DockstoreTool manualRegisterAndPublish(ContainersApi containersApi, String namespace, String name, String toolName,
```

Comment:
Nit: you don't need `public` for this and the other methods you moved here because you're only accessing this from inherited classes. As a general rule of thumb, specify the least amount of privilege needed.

That said, this is "just" a test, so I don't feel super strongly about it.